jaxley/synapses/tanh_conductance.py

```diff
@@ -0,0 +1,52 @@
+# This file is part of Jaxley, a differentiable neuroscience simulator. Jaxley is
+# licensed under the Apache License Version 2.0, see <https://www.apache.org/licenses/>
+
+from typing import Dict, Optional
+
+import jax.numpy as jnp
+
+from jaxley.synapses.synapse import Synapse
+
+
+class TanhConductanceSynapse(Synapse):
+    """
+    Compute synaptic current for a simple conductance synapse (no state). This is just
```

Comment:
The first line is what is being displayed in the API docs [here](https://jaxley.readthedocs.io/en/latest/jaxley.html#synapses). Please move the second sentence to a new line (and leave a blank line in between).